FinderKeeper/Assets/Scripts/CameraSwap.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraSwap : MonoBehaviour
+ {
+     public Camera IsometricCamera, ThirdPersonCamera;
+     public bool swapFlag = false;

```

Comment:
Just being picky, but it's not clear what value "swapFlag" implies from the name. Maybe a name like "isIsoCamera" would be clearer?

Reply:
Half the time the custom map just shows up as the black loading tiles, the other half it is extremely fuzzy. So I just changed the drop down from custom to a default style for testing purposes. Will need to investigate further.

Reply:
That's fair. I'll think of a clearer name for it